pytest_rpc/helpers.py

```diff
@@ -322,15 +331,14 @@ def _resource_in_list(service_type, service_name, expected_resource, run_on_host
 
     for i in range(0, retries):
```

Comment:
[There is a PyPI package for retrying methods/functions using a decorator that might interest you.](https://github.com/jd/tenacity)